.github/workflows/nodejs.yml

```diff
@@ -28,6 +28,9 @@ jobs:
         with:
```

Comment:
Reminds me I was going to work on a reusable workflow to install npm dependencies safely and in isolation. 
Are we sure this is using our config to ignore scripts?
Not necessary in the current PR, but we need to revisit the approach to installing dependencies

Reply:
It just runs `npm ci --foregbround-scripts`, so it does whatever that does.  It gives us some caching otherwise.